runtime/tutorials/how_to_with_npm/prisma.md

```diff
@@ -27,7 +27,7 @@ cd rest-api-with-prisma-oak
 Then, let's run `prisma init` with Deno:
 
 ```shell, ignore
-deno run --allow-read --allow-env --allow-write npm:prisma@^4.5 init
+deno run --allow-read --allow-env --allow-write npm:prisma@latest init
```

Comment:
Please keep this pinned to a recent version.

Reply:
I recommend sticking to the latest version, which we release every 3 weeks, as it contains the latest features and improvements. We also test every version to ensure it works as it should with Deno and other platforms/ libraries/ frameworks in our [ecosystem-tests repo](https://github.com/prisma/ecosystem-tests).🙂